database-mongo/load-mongo.js

```diff
@@ -0,0 +1,115 @@
+const db = require('./index.js');
+const mongoose = require('mongoose');
+const faker = require('faker');
+const moment = require('moment');
+
+var reviewSchema = new mongoose.Schema({
+  _id: {
+    type: Number,
+    unique: true,
+    index: true
+  },
+  review_date: Date,
+  review_text: String,
+  accuracy: Number,
+  communication: Number,
+  cleanliness: Number,
+  location: Number,
+  checkin: Number,
+  u_id: Number,
+  username: String,
+  display_name: String,
+  photo_url: String,
+  profile_url: String
+}, {autoIndex: false})
+
+var listingSchema = new mongoose.Schema({
+  _id: {
+    type: Number,
+    unique: true,
+    index: true
+  },
+  reviews: [reviewSchema]
+}, {autoIndex: false})
+
+var Listing = mongoose.model('listing', listingSchema);
+var Review = mongoose.model('Review', reviewSchema);
+
+const randomDate = (startDate = new Date(2015, 08, 01), endDate = new Date()) => {
+  let rand = new Date(startDate.getTime() + Math.random() * (endDate.getTime() - startDate.getTime()));
+  return rand.toISOString().slice(0,10);
+}
+
+let startWrite;
+let endWrite;
+
+(async () => {
+  let startAll = moment();
+
+  let listingIdTracker = 1;
+  let reviewCounter = 1;
+
+  for (let i = 0; i < 500; i++) {
+    let listingsArr = [];
+
+    startWrite = moment();
+    
+    for (let j = 0; j < 2000; j++, listingIdTracker++) {
```

Comment:
Personally, I think just renaming 'j' to be 'listingIdTracker' would make this quicker and easier to read and understand. Up to you.

Reply:
`j` and `listingIdTracker` are different since `j` resets every 2,000 entries, whereas `listingIdTracker` never resets and is how I am assigning a unique `_id` for each record.

Reply:
Got it. Fine as is then.